frontend/package.json

```diff
@@ -2,10 +2,20 @@
   "name": "frontend",
   "version": "0.1.0",
   "private": true,
+  "config": {
+    "input": "src/",
+    "intermediate": "tmp/",
+    "output": "dist/"
+  },
   "scripts": {
-    "start": "parcel serve src/!(layout).html --port 3000 --dist-dir .serve",
-    "build": "parcel build \"src/!(layout).html\"",
-    "lint": "eslint ."
+    "11ty:build": "eleventy --input=src/ --output=tmp/",
+    "11ty:watch": "eleventy --input=src/ --output=tmp/ --watch",
+    "build": "run-s -l clean 11ty:build parcel:build",
+    "clean": "rimraf dist/ tmp/; mkdir -p dist/ tmp/",
```

Comment:
Let's better use the `rm -rf` command instead of an extra dependecy.

Reply:
this is done.